stac_fastapi/core/stac_fastapi/core/core.py

```diff
@@ -375,8 +371,9 @@ async def all_collections(self, username_header: dict, **kwargs) -> Collections:
             ):
                 # Get access control array for each collection
                 try:
-                    access_control = collection["access_control"]
-                    collection.pop("access_control")
+                    access_control = collection["access_control_workspaces"]
+                    collection.pop("access_control_owner")
+                    collection.pop("access_control_workspaces")
```

Comment:
The knowledge that these fields must be removed is distributed in over a dozen places - can this not be centralized? For example, could serializers.py remove these fields (or all fields starting `__internal_` or `__` or `__access_control_`) in the `db_to_stac` methods?

Reply:
Moved to serializers